### ISSUE TITLE
brew

### ISSUE THREAD

**Issue by loeffel-io** [jarulraj/sqlcheck#31]
```bash
==> Tapping gwerbin/tap
Cloning into '/usr/local/Homebrew/Library/Taps/gwerbin/homebrew-tap'...
remote: Enumerating objects: 409, done.
remote: Total 409 (delta 0), reused 0 (delta 0), pack-reused 409
Receiving objects: 100% (409/409), 65.59 KiB | 197.00 KiB/s, done.
Resolving deltas: 100% (143/143), done.
Error: Invalid cask: /usr/local/Homebrew/Library/Taps/gwerbin/homebrew-tap/Casks/sqleo.rb
Cask 'sqleo' is unreadable: undefined method `method_missing_message' for Utils:Module
Error: Invalid cask: /usr/local/Homebrew/Library/Taps/gwerbin/homebrew-tap/Casks/squirrelsql.rb
Cask 'squirrelsql' is unreadable: undefined method `method_missing_message' for Utils:Module
Error: Invalid cask: /usr/local/Homebrew/Library/Taps/gwerbin/homebrew-tap/Casks/yiddish-klal.rb
Cask 'yiddish-klal' is unreadable: undefined method `method_missing_message' for Utils:Module
Error: Cannot tap gwerbin/tap: invalid syntax in tap!
```

**Comment by miketheman** [comment]
Seems like this issue should be reported to https://github.com/gwerbin/homebrew-tap - not here.

**Comment by dzsquared** [comment]
closing as this is an unrelated issue
